system/jlib/jstream.hpp

```diff
@@ -83,6 +83,9 @@ interface IBufferedSerialOutputStream : extends ISerialOutputStream
     virtual void suspend(size32_t wanted) = 0;   // Reserve some bytes and prevent data being flushed to the next stage until endNested is called.  May nest.
     virtual void resume(size32_t len, const void * ptr) = 0;  // update the data allocated by suspend and allow flushing.
     virtual void replaceOutput(ISerialOutputStream * newOutput) = 0;
+    virtual void append(const char * value) = 0;
```

Comment:
This interface needs to be as clearn/tight as possible.  Please implement them as global helpers instead.